src/components/about/Subteam.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Image from "next/image";
+import Placeholder from "@/public/images/placeholder.webp";
+import SubteamTag from "../SubteamTag";
+
+const Subteam = ({ text }) => {
+  return (
+    <div>
+      <div className="pb-3 max-w-fit">
+        <>
+          <SubteamTag text={"SUBTEAM"} />
+        </>
+      </div>
+      <div className="flex items-center">
+        <Image
+          src={Placeholder}
+          alt="Placeholder"
+          className="size-44 rounded-full float-left"
```

Comment:
You should use a fraction or percentage instead of 44 directly (that is hardcoding the pixels for a big component which won't scale well)